bitnami/kibana/README.md

```diff
@@ -125,6 +125,7 @@ The command removes all the Kubernetes components associated with the chart and
 | `service.nodePort`                     | Specify the nodePort value for the LoadBalancer and NodePort service types                                                                                | `""`                     |
 | `service.externalTrafficPolicy`        | Enable client source IP preservation                                                                                                                      | `Cluster`                |
 | `service.annotations`                  | Annotations for Kibana service (evaluated as a template)                                                                                                  | `{}`                     |
+| `service.labels`                  | Labels for Kibana service                                                                                                  | `{}`                     |
```

Comment:
Did you update the table using the [readme-generator tool](https://github.com/bitnami-labs/readme-generator-for-helm)? It looks like it's not correctly aligned. Could you take a look?

Reply:
Before it was not generated by that tool. I've just updated it through above mentioned readme-generator.